torch/ao/quantization/_quantize_pt2e.py

```diff
@@ -70,7 +71,7 @@ def prepare_pt2e_quantizer(
     model = prepare(
         model,
         quantizer,
-        False,  # is_qat
+        is_qat,
```

Comment:
wait, how does this work? in L70 we already run _fuse_conv_bn_ for PTQ

Reply:
I guess that is not run beacuse batchnorm has a different form in training

Reply:
Right I meant to put `fuse_conv_bn` under `if not qat` but it got lost in refactoring

Reply:
Right I meant to put `fuse_conv_bn` under `if not qat` but it got lost in refactoring. I'll separate prepare_qat out

Reply:
Oh yeah I meant to put `fuse_conv_bn` under `if not is_qat` but it got lost in refactoring